Linux Script/configure-linux.sh

```diff
@@ -875,7 +876,7 @@ searchAndFetch()
 {
 	url=$2
 
-	result=$(wget -qO- /dev/null --user "$LOGGLY_USERNAME" --password "$LOGGLY_PASSWORD" "$url")
+	result=$(curl -s -u $LOGGLY_USERNAME:$LOGGLY_PASSWORD $url)
```

Comment:
We have had curl built into our script for a long time and haven't heard any complaints. I'm guessing it's common or easy to install.